### PR TITLE
README improvements

### PR DESCRIPTION
**Pull request by ellm**
Small fixes to the README files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
+* [Web](src/web/README.md) - General information on browser script for inserting/replacing markup. Source code is further documented via jsdoc
+* [Dist](dist/README.md) - General information on the distribution (build) directory.
 
 ## Installation
- * Huron is not yet registered in NPM, so for now you'll have to check it out add it to your `package.json` via `file:./huron` or use a github link.
- * Don't forget to run `npm install` once you're done!
- * Once `npm install` finishes, you'll need to write a configuration or use the default. The default settings and how to modify them is documented in the configuration section of this readme.
- * Start the CLI using the `dist/cli/huron-cli.js` file.
- * Once the CLI has started, if you didn't run the CLI with the `--production` flag, you can access your prototype(s) at `localhost:[huron.port]/[huron.root]/[prototype-name].html`. You can find the defaults for the `port` and `root` options and how to change them in the config documentation, but using the defaults this URL would resolve to `localhost:8080/dist/index.html`.
+* Huron can be installed via [NPM](https://www.npmjs.com/package/huron). Add Huron to your `package.json` file.
```
Comment (ellm) [inline README.md]: I updated this part to reflect the recent addition to NPMJS.com

File: README.md [hunk 2]
```diff
@@ -1,34 +1,34 @@
...
- * [CLI](src/cli/README.md) - General information on the Huron CLI. Source code is further documented via jsdoc
- * [Web](src/web/README.md) - General infomration on browser script for inserting/replacing markup. Source code is further documented via jsdoc
- * [Dist](dist/README.md) - General information on the distribution (build) directory.
+* [Configuration](config/README.md) - Configuration for Huron and for your local webpack setup
+* [Templates](templates/README.md) - Peripheral template documentation
+* [CLI](src/cli/README.md) - General information on the Huron CLI. Source code is further documented via jsdoc
+* [Web](src/web/README.md) - General information on browser script for inserting/replacing markup. Source code is further documented via jsdoc
+* [Dist](dist/README.md) - General information on the distribution (build) directory.
 
 ## Installation
```
Comment (pattiereaves) [inline README.md]: +1 to that idea. Maybe make a ticket so we don't forget?

File: README.md [hunk 3]
```diff
@@ -1,34 +1,34 @@
...
- * Huron is not yet registered in NPM, so for now you'll have to check it out add it to your `package.json` via `file:./huron` or use a github link.
- * Don't forget to run `npm install` once you're done!
- * Once `npm install` finishes, you'll need to write a configuration or use the default. The default settings and how to modify them is documented in the configuration section of this readme.
- * Start the CLI using the `dist/cli/huron-cli.js` file.
- * Once the CLI has started, if you didn't run the CLI with the `--production` flag, you can access your prototype(s) at `localhost:[huron.port]/[huron.root]/[prototype-name].html`. You can find the defaults for the `port` and `root` options and how to change them in the config documentation, but using the defaults this URL would resolve to `localhost:8080/dist/index.html`.
+* Huron can be installed via [NPM](https://www.npmjs.com/package/huron). Add Huron to your `package.json` file.
+* Don't forget to run `npm install` once you're done!
```
Comment (pattiereaves) [inline README.md]: @ellm did you still need to `npm install` even with the new dist system?

File: README.md [hunk 4]
```diff
@@ -53,18 +53,18 @@ KSS is a documentation syntax and styleguide generator. All documentation should
...
  * You may include any valid HTML markup in your prototype.
  * Huron uses its own custom syntax for inserting templates from your KSS. This is accomplished via three data attributes:
  	* `[data-huron-id]` - The KSS styleguide section reference URI containing the template you want. As described above, to get the reference URI you simply convert any `.` to `-` in the section reference you wrote in the KSS. For example, the section `header.navigation` would be inserted using a `huron-id` of `header-navigation`.
  	* `[data-huron-type]` - This attribute roughly corresponds to the KSS field you want to insert. There are several options for this:
  		* `template` - This is the type you'll use most frequently, and will insert anything in your KSS `markup` field.
  		* `description` - Type to insert the KSS description field
- 		* `prototype` - This is mostly used by the HTML webpack plugin to insert your prototype content, but you can use it as well if necessary.
+ 		* `prototype` - This is mostly used by the HTML webpack plug-in to insert your prototype content, but you can use it as well if necessary.
```
Comment (pattiereaves) [inline README.md]: Plugin (not hyphenated) is how they spell it in the webpack docs: https://webpack.github.io/docs/plugins.html

**Comment by pattiereaves** [comment]
🦄

**Comment by ostowe** [comment]
lgtm, thanks @ellm!

File: README.md (hunk 3 above)
Comment (ellm) [inline README.md]: @pattiereaves Yeah, I did. Although, it's possible that I'm missing something. I'm still working on getting it up and running.

File: README.md (hunk 4 above)
Comment (ellm) [inline README.md]: D'oh, ty. That's what I get for following a spell checker :) Will fix that before merging

**Pull request merged**